docs/integration.rst

```diff
@@ -17,3 +17,38 @@ Here is an example, to add in your .pre-commit-config.yaml
     hooks:
       - id: yamllint
         args: [-c=/path/to/.yamllint]
+
+Integration with GitHub Actions
+-------------------------------
+
+Starting with version XXX of yamllint, it will automatically use the
```

Comment:
This needs a version number or to be rephrased to not specify when the feature was introduced

Reply:
Including a version number will make maintenance harder, and won't be much useful once the version is released and installed. I prefer not including it.

About the wording, I propose a small simplification, what do you think?
```
yamllint auto-detects when it's running inside of `GitHub
Actions<https://github.com/features/actions>` and automatically uses the suited
output format to decorate code with linting errors automatically. You can also
force the GitHub Actions output with ``yamllint --format github``.
```